src/main/java/ulcrs/data/DataFetch.java

```diff
@@ -47,34 +47,51 @@
 
         try
         {
+            // Note:
+            // The below commented code is the code required to hit the ULC endpoint to get ULC data.
+            //
+            // For now, we are accessing a json file on the ULC server directly to get this ULC data since this file
+            // has the necessary course requirement information, while the endpoint does not yet.
+
+//            // Initialize HTTP request
+//            URL url = new URL(DROP_IN_REQUEST_URL);
+//            connection = (HttpURLConnection) url.openConnection();
+//            connection.setRequestMethod(DROP_IN_REQUEST_REQUEST_TYPE);
+//            connection.setRequestProperty(CONTENT_TYPE, DROP_IN_REQUEST_CONTENT_TYPE);
+//            connection.setRequestProperty(COOKIE, cookie);
+//            connection.setRequestProperty(CONTENT_LENGTH, Integer.toString(DROP_IN_REQUEST_DATA.getBytes().length));
+//            connection.setRequestProperty(CONTENT_LANGUAGE, DROP_IN_REQUEST_CONTENT_LANGUAGE);
+//            connection.setUseCaches(false);
+//            connection.setDoOutput(true);
+//
+//            // Send request
+//            DataOutputStream wr = new DataOutputStream(connection.getOutputStream());
+//            wr.writeBytes(DROP_IN_REQUEST_DATA);
+//            wr.close();
+
             // Initialize HTTP request
-            URL url = new URL(DROP_IN_REQUEST_URL);
+            URL url = new URL("https://dropin.engr.wisc.edu/services/ulcrs-requirements.json");

```

Comment:
This could be made a constant, and the "GET" below. Since it is a temporary solution, it depends on how we want to handle it though

Reply:
Made constants